Source/JTSImageViewController.h

```diff
@@ -36,6 +36,11 @@ typedef NS_OPTIONS(NSInteger, JTSImageViewControllerBackgroundOptions) {
     JTSImageViewControllerBackgroundOption_Blurred = 1 << 1,
 };
 
+typedef NS_ENUM(NSInteger, JTSImageViewControllerDismissMode) {
+    JTSImageViewControllerDismissMode_UpDownDirection,
+    JTSImageViewControllerDismissMode_AnyDirection
+};
```

Comment:
Nice addition. I'd like to quibble with the name a bit. "Dismiss mode" sounds too broad for a feature that's just determining the allowed directions for flicking-to-dismiss (there are other ways to dismiss that aren't disallowed). What about renaming "dismiss mode" to "flick axis":

```
typedef NS_ENUM(NSInteger, JTSImageViewControllerFlickAxis) {
     JTSImageViewControllerFlickAxis_Vertical,
     JTSImageViewControllerFlickAxis_Any
 };
```

And rename the `modalDismissMode` property to `permittedDismissalFlickAxis`.